app.py

```
@@ -290,61 +399,150 @@ async def split():
                             """
-                    elif track1["isrc"] == track2["isrc"] and track1["commontrack_id"] == track2["commontrack_id"]:
-                        message = "Can not be splitted as they have the Same ISRC"
+                    elif (
+                        track1["isrc"] == track2["isrc"]
+                        and track1["commontrack_id"] == track2["commontrack_id"]
+                    ):
+                        message = _("Can not be splitted as they have the Same ISRC")
                     else:
-                        message = "They have different Pages"
+                        message = _("They have different Pages")
                 except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)